Lib/test/test_pty.py

```diff
@@ -15,20 +15,12 @@
 import socket
 import io # readline
 import unittest
-
-import struct
-import fcntl
```

Comment:
there's a test.support.import_helper.import_module("fcntl") on line 6 as well to skip the test if it doesn't exist, does this test have reason to care about fcntl existing at all now?

Reply:
Good catch.

Some context: I was the one who added the original `import struct` and `import fcntl` to define makeshift `getwinsize` and `setwinsize` type functions. But now that we have `termios.tcgetwinsize()` and `termios.tcsetwinsize()`, we no longer need those.

Answer to your original question: I think @tiran added the `import_module("fcntl")` on line 6 in the following commit to skip these tests on `wasm32-emscripten`: https://github.com/python/cpython/commit/c02e860ee79f29905be6fca997c96bb1a404bb32

I will remove that `import_module("fcntl")` line now.

Reply:
sadly the wasm32 tests crash on this test, so i've restored the line with a comment explaining why for now.